Adding issue template and code analysis github action (#702)
Azure/azure-storage-fuse: .github/ISSUE_TEMPLATE.md

> ### Which version of the blobfuse was used ?

Reviewer: ```suggestion
### Which version of blobfuse was used ?
```

Author: corrected

> ### Which version of the blobfuse was used ?

Reviewer 2: ```suggestion
### Which version of blobfuse was used?
```

Author: corrected

> ### Which OS (please include version) are you using ?

Reviewer 2: ```suggestion
### Which OS distribution and version are you using?
```

### If relevant, please share your mount command.

> ### What problem was encountered ?

Reviewer 2: ```suggestion
### What was the issue encountered?
```

> ### Have you found a mitigation/solution ?

Reviewer 2: ```suggestion
### Have you found a mitigation/solution?
```

> ### Kindly share logs if available.

Reviewer: ```suggestion
### Please share logs if available.
```
